Amazon.QLDB.Driver/result/Result.cs

```diff
@@ -182,19 +182,29 @@ public void Reset()
             /// </summary>
             ///
             /// <returns>The current IOUsage statistics.</returns>
-            internal IOUsage GetConsumedIOs()
+            internal IOUsage? GetConsumedIOs()
             {
-                return new IOUsage(this.readIOs, this.writeIOs);
+                if (this.readIOs != null || this.writeIOs != null)
```

Comment:
I'm not fully against this, but I worry about what happens if we get some odd case where readIOs is null, but writeIOs isn't, and then we call .Value on it. What happens in this case?

Reply:
That couldn't be the case. Both `readIOs` and `writeIOs` are set together. If `statementResult` has a `consumedIOs` then both of these value will have a 'long' value, if `statementResult` does not have any `consumedIOs` then both will be null. (Line 118)

Reply:
I understand that, hence why I said i'm not fully against it. 

I'm saying that this is concerning because it's code that could throw an exception and only doesn't because of a bunch of assumptions. What happens if someone else changes the code you're assuming to prevent this in the future?